.github/workflows/build.yml

```diff
@@ -88,7 +89,8 @@ jobs:
         toolchain: stable
         override: true
     - name: Run tests
-      run: cargo test --verbose --features datafusion-ext,azure
+      #run: cargo test --verbose --features datafusion-ext,azure TODO Revert back!
```

Comment:
@houqp fyi, we're turned off datafusion from build since it has deps on arrow that are not matching the hash that we have. Basically because we're on arrow-6.0.0-SNAPSHOT and datafusion is on 5.3.0

Reply:
I've tried to clone datafusion and reference the arrow rs deps to the main branch but has no success because of several tests failing

Reply:
yeah, that's fine. datafusion is not ready for 6.x yet, see https://github.com/apache/arrow-datafusion/pull/984

Reply:
So what's yout take on that? Should we nerge it as is with datafusion commented out?

Reply:
yes, i am fine with temporarily disable datafusion integration for now.